.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore=B008,E121,E123,E126,E203,E226,E24,E704,W503,W504,D100,D105,D200,D301,D402
+ignore=B008,E121,E123,E126,E203,E226,E24,E704,E731,W503,W504,D100,D105,D200,D301,D402
```

Comment:
Can you please suppress it in one particular case instead

Reply:
sure !